week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,9 +1,13 @@
 // Declare your function first - call it "add"
-
+function add() {
+  return 124 + 13;
+}
 // Call the function and assign to a variable `sum`
 
```

Comment:
Same comments apply here as in exercise K-1, make use of function parameters to return a dynamic result.